mlir/lib/ExecutionEngine/ROCm/rocm-runtime-wrappers.cpp

```diff
@@ -499,8 +499,8 @@ mcpuMemset5DFloatRandInt(float *allocated, float *aligned, int64_t offset,
                          int64_t size0, int64_t size1, int64_t size2,
                          int64_t size3, int64_t size4, int64_t stride0,
                          int64_t stride1, int64_t stride2, int64_t stride3,
-                         int64_t stride4, short min, short max, int seed) {
-  if (seed < 0)
+                         int64_t stride4, short min, short max, uint32_t seed) {
```

Comment:
I think to do it 100% right we need to populate the `ui32` type instead of `i32` type in the `mlir-miopen-driver.cpp` end. But `i32` does not concern about signed-ness so is correct too.